src/components/app.scss

```diff
@@ -132,6 +132,57 @@
     margin: auto;
   }
 
+  // split-pane css
+  .Resizer {
+    background: #000;
```

Comment:
Can you remove this background (as it introduces visual imbalance in our UI)? 

![voyager_2_and_vega-lite__git_](https://user-images.githubusercontent.com/111269/31857755-17dbe44a-b710-11e7-9f78-7073da73d2fb.png)